src/main/java/com/cburch/logisim/fpga/data/MapComponent.java

```diff
@@ -191,7 +191,8 @@ public boolean isMapped(int pin) {
   public boolean isInternalMapped(int pin) {
     if (pin < 0 || pin >= nrOfPins) return false;
     return isBoardMapped(pin)
-        && maps.get(pin).getIoComp().getType().equals(IoComponentTypes.LedArray);
+        && (maps.get(pin).getIoComp().getType().equals(IoComponentTypes.LedArray) 
+        || maps.get(pin).getIoComp().getType().equals(IoComponentTypes.SevenSegmentScanning));
```

Comment:
Check the logic here. You are calling maps.get(pin) in the right side of the 'or' when isBoardMapped(pin) is false. Is that what you intend to do?

Reply:
I think the confusion is wrong identation, the `or` is inside brackets. The brackets with an and with isBoardMapped(pin). Let me know if this is the case

Reply:
Ah. You are correct. Indenting the || line might help.

Reply:
Thanks! Done